src/components/elementView/elementTable.tsx

```diff
@@ -512,7 +512,8 @@ class ElementTable extends React.Component<
         col !== "comment" &&
         col !== "power_connections" &&
         col !== "mac_addresses" &&
-        col !== "network_connections"
+        col !== "network_connections" &&
+        col !== "is_admin"
```

Comment:
made it so we don't show is_admin column in the table. this is a bool so its values never appear in the column. I think it's redundant anyway since the grant/revoke admin button right next to it makes it obvious what the permission is currently

Reply:
agree